bin/test-workflow/cleanup_namespaces.sh

```diff
@@ -2,5 +2,12 @@
 
 source ./utils.sh
 
-$cli delete namespace "$CONJUR_NAMESPACE_NAME"
-$cli delete namespace "$TEST_APP_NAMESPACE_NAME"
+function delete_namespace {
+namespace=$1
```

Comment:
Should be `namespace="$1"`

Reply:
will replace with the --ignore-not-found option